src/explore-education-statistics-frontend/src/modules/find-statistics/components/__tests__/SearchForm.test.tsx

```diff
@@ -0,0 +1,35 @@
+import SearchForm from '@frontend/modules/find-statistics/components/SearchForm';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import noop from 'lodash/noop';
+import userEvent from '@testing-library/user-event';
+
+describe('SearchForm', () => {
+  test('renders correctly', () => {
+    render(<SearchForm onSubmit={noop} />);
+    expect(screen.getByLabelText('Search')).toBeInTheDocument();
+    expect(screen.getByRole('button', { name: 'Search' })).toBeInTheDocument();
+  });
+
+  test('renders correctly with an initial search term', () => {
+    render(<SearchForm searchTerm="find me" onSubmit={noop} />);
+    expect(screen.getByLabelText('Search')).toHaveValue('find me');
+  });
+
```

Comment:
Would suggest adding a test case where the validation error is removed once the input becomes valid. We should be a little more rigorous here as we're not backed up by Formik automatically handling stuff like this for us.